driver/mounter.go

```diff
@@ -156,7 +157,7 @@ func (m *mounter) Mount(source, target, fsType string, opts ...string) error {
 	mountArgs = append(mountArgs, target)
 
 	// create target, os.Mkdirall is noop if it exists
-	err := os.MkdirAll(target, 0750)
+	err := os.MkdirAll(target, 0o750)
```

Comment:
Can't find any reference regarding this file permission style. Is it related to symbolic permissions like `ugoa`?

Reply:
That's really interesting. For some reason, my GoLand seems to do it automatically. It's an indicator for octal number that follows and it seems to be a fairly new thing. We'd probably want to start using that, assuming it works with `os.MkdirAll`.

Reply:
Oh, similar to binary 0b and hex 0x then